client/src/components/DisplayTable/DisplayTable.js

```diff
@@ -93,6 +94,7 @@ export class DisplayTable extends React.Component {
           sum_func: (sum, value) => ... <- (function) Custom sum func. Default to sum + value
           sort_func: (a, b) => ... <- (function) Custom sort func. Default to _.sortBy
           sum_initial_value: 0 <- (number) Default to 0
+          is_toggleable: true <- (boolean) Default to true
```

Comment:
I think more explicit name should be used like `visibility_toggleable` or something like that.
Also, please checkout `column_config_defaults`. You shouldn't need to set this flag to `false` everywhere. Setting it `false` in defaults variable will do it.

I'll review the rest of the code after you make this change!